src/cpuid/src/brand_string.rs

```diff
@@ -300,6 +293,13 @@ impl BrandString {
     }
 }
 
+fn null_terminator_index(slice: &[u8]) -> usize {
```

Comment:
Nit: Another option would be to call this method `strlen` since I think it's equivalent to `strlen` from C.

Reply:
I'm a fan of overly-descriptive function names, but I can leave a comment about this if that's a fair compromise?
i'd argue that naming it strlen could have people unfamiliar with libc thinking they could swap this out with str::len

Reply:
Agree. I think a comment describing more precisely what the function is doing would help.